core/src/main/scala/org/apache/spark/ContextCleaner.scala

```diff
@@ -276,6 +287,16 @@ private[spark] class ContextCleaner(
     }
   }
 
+  def doCleanSparkListener(listener: SparkListener): Unit = {
+    try {
+      logDebug(s"Cleaning Spark listener $listener")
```

Comment:
Does logging `$listener` show meaningful message? It is not like other cases like clean up RDD (rdd id), or clean up broadcast (broadcast id).

Reply:
It logs the qualified class name and JVM object id of that listener, e.g.,

> Cleaning Spark listener org.apache.spark.sql.util.ExecutionListenerBus@54600fe

I think it's enough for users to know which listener is removed.

Others log their own ids because users can track them by analyzing the logs since Spark logs those ids in other places too. But Spark doesn't do that for listeners (and the listener itself doesn't have such an identifier or something else) so it should be enough for now.